src/components/App/App.js

```diff
@@ -1,22 +1,50 @@
 import './App.css';
 import Header from '../Header/Header';
 import MovieCardContainer from '../MovieCardContainer/MovieCardContainer';
+import Error from '../Error/Error';
 import SingleMovie from '../SingleMovie/SingleMovie';
 import { useState, useEffect } from 'react';
 
 function App() {
   const [movieData, setMovieData] = useState([]);
   const [showMovie, setShowMovie] = useState(false);
-  const [clickedMovie, setClickedMovie] = useState([])
+  const [clickedMovie, setClickedMovie] = useState([]);
+  const [error, setError] = useState(false);
 
   useEffect(() => {
     fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
```

Comment:
You may have already done this since this PR is from last week but consider pulling this fetch functionality out into a function declared in an apiCalls file.  Then you can import and invoke that function in the useEffect here